index.js

```diff
@@ -100,6 +100,7 @@ function send(ctx, path, opts) {
     ctx.set('Cache-Control', 'max-age=' + (maxage / 1000 | 0));
     ctx.type = type(path);
     ctx.body = fs.createReadStream(path);
+    ctx.length = stats.size;
```

Comment:
I squashed the commits but isn't it possible that removing ctx.set('Content-Length') will break current uses?